internal/controller/kyma_controller.go

```diff
@@ -131,6 +133,7 @@ func (r *KymaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 			r.deleteRemoteClientCache(ctx, kyma)
 			r.enqueueWarningEvent(kyma, syncContextError, err)
```

Comment:
before this enqueueWarningEvent should also have a reason, right?

Reply:
This is the same error as the SyncContextRetrievalError, the requeue will happen in line 137